File: Iniciante/URI 1008.pas
var
    funcionarios : int64;
    horas : int64;
	valorPorHora : double;
	salario : double;

begin
    read(funcionarios);
    read(horas);
	read(valorPorHora);
	
    salario := horas * valorPorHora;
    
	writeln('NUMBER = ', funcionarios);
	writeln('SALARY = U$ ', salario:0:2);
end.

File: Iniciante/URI 1008.clj
(def funcionarios (read-string (read-line)))
(def horas (read-string (read-line)))
(def valorPorHora (read-string (read-line)))
(def salario (* horas valorPorHora))

(printf "NUMBER = %d\n" funcionarios)
(printf "SALARY = U$ %.2f\n" salario)